php/blocks/class-loader.php

```diff
@@ -434,7 +423,7 @@ public function block_template( $name, $type = 'block' ) {
 				'<div class="notice notice-warning">%s</div>',
 				wp_kses_post(
 					// Translators: Placeholder is a file path.
-					sprintf( __( 'Template file %s not found.' ), '<code>' . esc_html( $template_file ) . '</code>' )
+					sprintf( __( 'Template file %s not found.' ), '<code>' . esc_html( $templates[0] ) . '</code>' )
```

Comment:
`$templates[0]` will probably always be set, unless there's a change to the codebase.

But what do you think about checking that `isset( $templates[0] )`, to future-proof this? Maybe in the conditional above:

```php
if ( ! current_user_can( 'edit_posts' ) || ! isset( $templates[0] ) ) {
	return;
}
```

Reply:
Great idea!

Reply:
Fixed in 06ad5a0.

Reply:
Thanks, that looks great.